dotnet/src/Connectors/Connectors.Anthropic/Core/ClientBase.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Http;
+
+namespace Microsoft.SemanticKernel.Connectors.Anthropic.Core;
+
+internal abstract class ClientBase
```

Comment:
Suggest Composition over Inheritance for the Client, this is how we do in `OpenAI` connector and since this one is starting I suggest using this client as a `ClientCore` instead of an abstract class.

Reply:
@RogerBarreto probably I will merge this class into one ClaudeClient class. But for initial PR leave as is.